test/integration/associations/belongs-to-many.test.js

```diff
@@ -3538,4 +3538,97 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), () => {
       expect(hat.hatwornbys[0].name).to.equal('Foo Bar');
     });
   });
+
+  describe('Disabling default unique constraint', () => {
+    beforeEach(function () {
+      this.Order = this.sequelize.define('order', {});
+      this.Good = this.sequelize.define('good', {
+        name: DataTypes.STRING,
+      });
+      this.OrderItem = this.sequelize.define('orderitem', {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+        },
+        isUpsell: {
+          type: DataTypes.BOOLEAN,
+          defaultValue: false,
+        },
+      });
+
+      this.Order.belongsToMany(this.Good, {
+        through: {
+          model: this.OrderItem,
+          unique: false,
+        },
+      });
+      this.Good.belongsToMany(this.Order, {
+        through: {
+          model: this.OrderItem,
+          unique: false,
+        },
+      });
+    });
+
+    it('should create new relations in parallel', async function () {
+      await this.sequelize.sync({ force: true });
+
+      const { Order, Good } = this;
+
+      const order = await Order.create();
+      const good = await Good.create({ name: 'Coca-Cola' });
```

Comment:
```suggestion
      const good = await Good.create({ name: 'Drink' });
```
Other suggestions are fine as well but I'd prefer not to include brand names